pkg/meta/model/reorg.go

```diff
@@ -105,9 +109,29 @@ func (dm *DDLReorgMeta) GetBatchSizeOrDefault(defaultVal int) int {
 
 // SetBatchSize sets the batch size in DDLReorgMeta.
 func (dm *DDLReorgMeta) SetBatchSize(batchSize int) {
+	if dm == nil {
+		dm = &DDLReorgMeta{}
+	}
 	atomic.StoreInt64(&dm.BatchSize, int64(batchSize))
 }
 
+// GetMaxWriteSpeedOrDefault gets the max write speed from DDLReorgMeta.
+// 0 means no limit.
+func (dm *DDLReorgMeta) GetMaxWriteSpeedOrDefault(defaultVal int) int {
+	if dm == nil {
+		return defaultVal
+	}
+	return int(atomic.LoadInt64(&dm.MaxWriteSpeed))
```

Comment:
use atomic.XXX type

Reply:
I guess the reason is `atomic.XXX` does not implement (un)marshal interface, so we must store the plain type in struct and use helper functions

Reply:
golang's doesn't, but uber's does

Reply:
If we use the atomic.XXX type, seems these three fields wiil meet unmarshal error when the job meta is from old version(upgrade)?